vlib/regex/README.md

```diff
@@ -250,14 +250,14 @@ The array will be filled with the following logic:
 The regex save until finish or found that the array have no space.
 If the space ends no error is raised, further records will not be saved.
 
-```v oksyntax
+```v ignore
 fn example2() {
 	test_regex()
 	text := 'tst: 01,23,45 ,56, 78'
 	query := r'.*:(\s*\d+[\s,]*)+'
 	mut re := new() or { panic(err) }
 	// re.debug = 2
-	re.group_csave = [-1].repeat(3 * 20 + 1) // we expect max 20 records
+	re.group_csave_flag = true  // enable continuos capture
```

Comment:
I suppose this is meant to be *continuous* 😉
This typo appears multiple times in the code.